system-contracts/SystemContractsHashes.json

```diff
@@ -3,49 +3,49 @@
     "contractName": "AccountCodeStorage",
     "bytecodePath": "artifacts-zk/contracts-preprocessed/AccountCodeStorage.sol/AccountCodeStorage.json",
     "sourceCodePath": "contracts-preprocessed/AccountCodeStorage.sol",
-    "bytecodeHash": "0x0100007549287362e4263ea5b204f01fc3c7f2ac09d71e6eb21029698220f01a",
+    "bytecodeHash": "0x01000075d58c2cea1d80b434fb70e256b90ba0fbe7b0bd1f5dc70350f7bdfd34",
```

Comment:
Why do we have all these system contract hashes changed, if we are only changing L1BaseToken contract?

Reply:
Yes, it was weird to me too. I think they did not update this code last time they updated the bootloader.
I followed the readme to compute the new hashes https://github.com/vianetwork/era-contracts/blob/main/system-contracts/README.md

Reply:
The reason is probably the change in IBaseToken interface.